pygfunction/api.py

```diff
@@ -0,0 +1,39 @@
+from pygfunction.gfunction import gFunction
+from pygfunction.boreholes import Borehole
+from typing import Dict, Optional, Any, List
+
+
+class PYG(object):
+
+    def __init__(self, borehole_config_list: List[List[float]],
+                 alpha: float,
+                 time: List[float],
+                 options: Optional[Dict[str, Any]] = None,
+                 solver_method: str = "equivalent",
+                 boundary_condition: str = "UHTR"):
+
+        """
+        Borehole config parameters are defined in the Borehole class
+        All other parameters are defined in the PyGFunction class
+        """
+
+        borefield = []  # List of borehole objects
+
+        if options is None:
+            options = {}
+
+        for borehole_config_items in borehole_config_list:
+            _borehole = Borehole(*borehole_config_items)
+            borefield.append(_borehole)
+
+        self.gfunc = gFunction(
+            borefield,
+            alpha,
+            time=time,
+            boundary_condition=boundary_condition,
+            options=options,
+            method=solver_method,
+        )
```

Comment:
@Myoldmopar responding to this comment (https://github.com/mitchute/pygfunction/pull/4/files#r1834563947) the gFunction class needs a list of Borehole class instances. All this is doing is setting up the __init__ arguments which that class needs. Those parameters are here: https://github.com/mitchute/pygfunction/pull/4/files#diff-3451cc474d5698222168f66a293339a6bd296ee2c56f56409866bbae99917a79R25

pygfunction does have some methods for building out these parameter lists for some GHE configurations, so maybe we can offload some of that there. For custom GHE configurations, I think we'll likely end up doing a lot of this ourselves.